sub.go

```diff
@@ -134,6 +136,8 @@ type Subscription struct {
 	Billing               SubscriptionBilling            `json:"billing"`
 	BillingCycleAnchor    int64                          `json:"billing_cycle_anchor"`
 	BillingThresholds     *SubscriptionBillingThresholds `json:"billing_thresholds"`
+	CancelAt              int64                          `json:"cancel_at"`
+	CancelAtPeriodEnd     bool                           `json:"cancel_at_period_end"`
```

Comment:
that one was not in the right alphabetical order :p

Reply:
Haha, good catch!

Reply:
Truthfully, I did not catch it. I noticed the field was missing, added it, looked at the original PR and then figured out what happened :p